Cinema/src/main/java/com/example/Cinema/model/Projections.java

```diff
@@ -1,30 +1,34 @@
 package com.example.Cinema.model;
 
-import com.example.Cinema.validator.*;
+import com.example.Cinema.validator.ValidIdHall;
+import com.example.Cinema.validator.ValidIdMovie;
+import com.example.Cinema.validator.ValidTime;
 
+import javax.validation.Valid;
 import javax.validation.constraints.FutureOrPresent;
 import javax.validation.constraints.NotNull;
 import java.time.LocalDate;
 import java.time.LocalTime;
 
-
+@Valid
 public class Projections {
```

Comment:
This class represents a single Projection, so the name shouldn't really be plural.

Reply:
Change all class names in singular. Created packages for response, request and db models. And refactored name of all classes